app/pages/_app.tsx

```diff
@@ -78,6 +76,7 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
         <BCGovTypography />
         <Sentry.ErrorBoundary fallback={<Error500 />}>
           <RelayEnvironmentProvider environment={env}>
+            {typeof window !== 'undefined' && <SessionExpiryHandler />}
```

Comment:
Not sure if this will do anything though I checked the `cas` repo and this is where they have it.